redisinsight/ui/src/utils/formatters/valueFormatters.tsx

```diff
@@ -108,16 +110,30 @@ const formattingBuffer = (
       }
     }
     case KeyValueFormat.Vector32Bit: {
+      const utfVariant = bufferToUTF8(reply)
       try {
+        if (props?.isField) return { value: utfVariant, isValid: true }
+        const bufferFromUtf = Buffer.from(utfVariant)
```

Comment:
you don't need again transform from utf to Boofer. `reply` - already a Buffer

Reply:
done